packages/form-js-viewer/src/features/repeatRender/RepeatRenderManager.js

```diff
@@ -79,11 +79,11 @@ export class RepeatRenderManager {
 
     return (
       <>
-        {displayValues.map((value, index) =>
+        {displayValues.map((itemValue, itemIndex) =>
           <RepetitionScaffold
```

Comment:
(nitpick, something to keep in mind for the future) 

I would have `{ ...restProps }` at the start do that it can't override any explicitly named props in future and help avoid this issue.

React will evaluate properties in order so you can define elements like so
```jsx
<SomeComponent
  overridableProperty={...}
  {...restProps}
  nonOverridableProperty={...}
/>